1-exercises/C-undefined-properties/exercise.js

```diff
@@ -14,7 +14,7 @@ let car = {
   yearsOld: 8,
 };
 
-console.log(car["colour"]);
+console.log(car["colour"]); //"colour" is not defined as an object anywhere in the variable statement//
```

Comment:
Please check the CYF lesson about objects: https://syllabus.codeyourfuture.io/js-core-2/week-1/lesson#objects